src/test/groovy/life/qbic/datamodel/datasets/datastructure/OxfordNanoporeExperimentSpec.groovy

```diff
@@ -28,6 +28,12 @@ class OxfordNanoporeExperimentSpec extends Specification {
      * according to the schema containing unclassified read information
      */
     @Shared
+    Map extendedWorkingSimpleDataStructureWithReportsFolder
```

Comment:
The description of `extendedWorkingSimpleDataStructureWithReportsFolder` and `unclassifiedWorkingDataStructure` is the same. 
I'd also suggest shortening the variable if possible (The extended and Simple keywords seem unnecessary). 
Nit: Also in both descriptions there is a duplicate "that"